### PR TITLE
Branch AK

### PR DESCRIPTION
**Pull request by FarukOzderim**
Moved the fork of @AK391 over here for ease of collab.

History of the PR: #27

### CODE REVIEW

**Review by osanseviero — APPROVED** [review]
LG!

**Comment by AK391** [comment]
@FarukOzderim no worries, I can still add them, added the change for the headers

File: gradio-blocks/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+3. Submitting your demo on Spaces to the Gradio Blocks Party Organization
+4. Share your blocks demo with a permanent shareable link 
+5. Win Prizes
+
+
+## Example spaces using Blocks
+
+<img width="1180" alt="mindseye-lite" src="https://user-images.githubusercontent.com/81195143/168619604-cf1ac733-c10e-487f-add4-8da48002dcff.png">
+
+- app: https://huggingface.co/spaces/dalle-mini/dalle-mini
```
[the remark this answers is not in the document]
Comment (AK391) [reply gradio-blocks/README.md]: thanks updated

**Review by nateraw — APPROVED** [review]
LGTM 🚀

**Comment by AK391** [comment]
@nateraw thanks merged

**Pull request merged**